shared/wallets/wallet-list/index.js

```diff
@@ -12,10 +19,11 @@ type AddProps = {
 
 const rowHeight = isMobile ? 56 : 48
 
-const styles = styleSheetCreate({
-  addContainerBox: {height: rowHeight, paddingTop: globalMargins.small},
-  progressIndicator: {height: 30, width: 30},
-})
+const HoverBox = isMobile
+  ? Kb.Box2
+  : glamorous(Kb.Box2)({
```

Comment:
note we dont need this anymore, there is hover css now

Reply:
`className='hover_background_color_blueGrey2'`

Reply:
Oh, hi. 👋